inc/modules/export/epub/templates/epub201/opf.php

```diff
@@ -45,6 +47,10 @@
 		echo '<dc:creator opf:role="aut"';
 		if ( ! empty( $meta['pb_author_file_as'] ) ) {
 			echo ' opf:file-as="' . $meta['pb_author_file_as'] . '"';
+		} else {
+			$nameparser = new Parser();
+			$author = $nameparser->parse( $meta['pb_author'] );
```

Comment:
This library [can throw exceptions](https://github.com/davidgorges/HumanNameParser.php/blob/master/src/HumanNameParser/Parser.php#L188). What happens if it's a bad name? Try/Catch?